var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -362,7 +362,6 @@ def patch_hdf5_pkgconfigcmake(self):
             return
 
         pkgconfig_file = find(self.prefix, "netcdf.pc", recursive=True)
-        cmakeconfig_file = find(self.prefix, "netCDFTargets.cmake", recursive=True)
         ncconfig_file = find(self.prefix, "nc-config", recursive=True)
         settingsconfig_file = find(self.prefix, "libnetcdf.settings", recursive=True)
 
```

Comment:
This variable seems to be used below (line 369), so this function seems incorrect now?

Reply:
L369 needs to be changed to `files = pkgconfig_file + ncconfig_file + settingsconfig_file`